### PR TITLE
PIMOB-1432: Revise country code list

### PR DESCRIPTION
**Pull request by maxim-nosov-cko**
## Issue

[PIMOB-1432: Revise country code list](https://checkout.atlassian.net/browse/PIMOB-1432)

## Proposed changes

Country list was updated according to https://www.checkout.com/docs/resources/codes/country-codes

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._

* [x] Reviewers assigned
* [x] I have performed a self-review of my code and manual testing
* [x] Lint and unit tests pass locally with my changes
* [ ] I have added tests that prove my fix is effective or that my feature works
* [ ] I have added necessary documentation (if applicable)

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you choose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: checkout/src/main/java/com/checkout/base/model/Country.kt [hunk 1]
```diff
@@ -29,8 +29,10 @@ public enum class Country(internal val iso3166Alpha2: String, internal val diali
     BERMUDA("BM", "1441"),
     BHUTAN("BT", "975"),
     BOLIVIA("BO", "591"),
+    BONAIRE("BQ", "5997"),
```
Comment (deepesh-vasthimal-cko) [inline checkout/src/main/java/com/checkout/base/model/Country.kt:32]: @maxim-nosov-cko Are they no unit test for these changes ?

**Review by deepesh-vasthimal-cko — APPROVED** [review]

File: checkout/src/main/java/com/checkout/base/model/Country.kt (hunk 1 above)
Comment (maxim-nosov-cko) [reply checkout/src/main/java/com/checkout/base/model/Country.kt:32]: @deepesh-vasthimal-cko 
No, there are no unit tests.
What kind of tests do you want to see for the enum class?

Comment (deepesh-vasthimal-cko) [reply checkout/src/main/java/com/checkout/base/model/Country.kt:32]: For iOS we have something like this : https://github.com/checkout/frames-ios/blob/7960a882dc25cf2f84ade3c519f916f32cda3c60/Checkout/Checkout/Test/Model/CountryTests.swift

Comment (maxim-nosov-cko) [reply checkout/src/main/java/com/checkout/base/model/Country.kt:32]: `test_Country_init` is not relevant to current changes, probably a test for fun from() could be written.

The purpose of `test_allAvailable_matchesExpected` is not clear to me at all. With the same success, tests can be written for all constant variables, i.e. value of `some_contant` equals the value of that constant.

In Android we use enum and function [`values()`](https://kotlinlang.org/api/latest/jvm/stdlib/kotlin/enum-values.html) which is a default Koltin functionality, returns array of all enum values. And as far as I'm concerned it's final and can't be even overridden.

Comment (maxim-nosov-cko) [reply checkout/src/main/java/com/checkout/base/model/Country.kt:32]: @deepesh-vasthimal-cko If you still want to see those tests on Android please let me know.

**Pull request merged**